lib/observer/observer.ex

```diff
@@ -0,0 +1,14 @@
+defmodule Observer do
+require Satellite.Constants
+  defstruct [:latitude, :longitude, :height]
+
+  def create_from(latitude_degrees, longitude_degrees, height_km) do
```

Comment:
Tried adding the macros, but it was giving my some fits. I'll try some more after I merge this thing.